src/index.html

```diff
@@ -1,12 +1,382 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head >
+    <meta charset="UTF-8" >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" >
+    <link rel="preconnect" href="https://fonts.googleapis.com" >
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.scss" >
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header page__header">
+      <div class="header__logo">
+        <a href="#" class="header__logoLink">
```

Comment:
```suggestion
        <a href="#" class="header__logo-link">
```
it's better to name with kebab-case